spec/v1/CSN-Interop-Effective.schema.yaml

```diff
@@ -859,11 +861,17 @@ definitions:
           - $ref: "#/definitions/DecimalScaleType"
         description: |-
           Describes the number of digits to the right of the decimal point in a number.
+
+          SHOULD be explicitly provided and MUST be provided if own default assumptions diverge from specified default of `floating`.
         default: "floating"
     patternProperties: *patternPropertiesAnnotationsAndPrivate
     required:
       - type
     additionalProperties: false
+    # Debatable if we want to create validation warnings by default.
```

Comment:
@maiargu pointed out that if we disable x-recommended here, it would be consistent to also do it for cardinality min and max